data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -332,6 +332,9 @@ then
 	if [ -f "$PWD/manifests/cloud-provider-config.yaml" ]; then
 		ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --cloud-config-file=/assets/config-bootstrap/cloud-provider-config-generated.yaml"
 	fi
+	if [ -f "$PWD/manifests/openshift-lbconfig-custom-dns.yaml" ]; then
```

Comment:
```suggestion
	if [ -f "/opt/openshift/manifests/openshift-lbconfig-custom-dns.yaml" ]; then
```

Reply:
@sadasu this is changed to "/opt/openshift/manifests/openshift-lb-config.yaml" here https://github.com/openshift/installer/pull/7631